app/views/main.scala.html

```diff
@@ -5,17 +5,14 @@
 <head lang="en">
     <meta charset="UTF-8">
     <title>Composer Restorer - @pageTitle</title>
-    <link rel="stylesheet" type="text/css" href="@routes.Assets.at("jspm_packages/github/guardian/scribe-plugin-noting@0.3.29/skins/gu-noting.css")" />
```

Comment:
This package depends on bower so won't even install in a fresh NPM project 🤦 

I've copy pasted the CSS into this project instead. Presumably this hasn't kept pace with print only/web only and other ProseMirror powered fun so is it a big deal?